files/jsondata/L10n-Template.json

```
@@ -72,19 +72,22 @@
     "fr": "Non standard",
     "ja": "非標準",
     "ko": "비표준",
+    "pt-BR": "Não padrão",
     "ru": "Не стандартно",
     "zh-CN": "非标准"
   },
 
   "non_standard_header_copy": {
-    "de": "Diese Funktion ist nicht standardisiert und befindet sich nicht auf dem Weg zur Standardisierung. Verwenden Sie sie nicht auf Produktionsseiten, die dem Web ausgesetzt sind: Sie funktioniert nicht für alle Benutzer. Es kann auch große Inkompatibilitäten zwischen Implementierungen geben, und das Verhalten kann sich in Zukunft ändern.",
-    "en-US": "This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.",
-    "es": "Esta característica no es parte de los estándares. No la uses en sitios Web en producción: no funcionará para todos los usuarios. Podrían haber también incompatibilidades considerables entre distintas implementaciones y el comportamiento podría cambiar en el futuro.",
-    "fr": "Cette fonctionnalité n'est ni standard, ni en voie de standardisation. Ne l'utilisez pas pour des sites accessibles sur le Web : elle ne fonctionnera pas pour tout utilisateur. Il peut également y avoir d'importantes incompatibilités entre les implémentations et son comportement peut être modifié dans le futur.",
-    "ja": "この機能は標準ではなく、標準化の予定もありません。公開されているウェブサイトには使用しないでください。ユーザーによっては使用できないことがあります。実装ごとに大きな差があることもあり、将来は振る舞いが変わるかもしれません。",
-    "ko": "이 기능은 비표준이므로 실제 프로덕션에서 사용하지 마세요. 모든 사용자 환경에서 작동하지 않을 수 도 있으며, 미래에 호환성 문제가 생길 수 있습니다.",
-    "ru": "Эта возможность не является стандартной и стандартизировать её пока никто не собирается. Не используйте её на сайтах, смотрящих во внешний мир: она будет работать не у всех пользователей. Также могут присутствовать большие несовместимости между реализациями и её поведение может в будущем измениться.",
-    "zh-CN": "该特性是非标准的，请尽量不要在生产环境中使用它！"
+    "de": "Diese Funktion ist nicht standardisiert. Wir raten davon ab, nicht-standardisierte Funktionen auf produktiven Webseiten zu verwenden, da sie nur begrenzt von Browsern unterstützt werden und sich ändern oder entfernt werden können. In bestimmten Fällen, in denen es keine standardisierte Alternative gibt, kann sie jedoch eine geeignete Option sein.",
```

@argl Since you already reviewed some German strings, do you have opinions here?

---

```suggestion
    "de": "Diese Funktion ist nicht standardisiert. Wir raten davon ab, diese Funktionen auf produktiven Webseiten zu verwenden, da sie nur von bestimmten Browsern unterstützt wird und sich in Zukunft ändern oder entfernt werden kann. Unter Umständen kann sie jedoch eine geeignete Option sein."
```

---

@argl Hier eine überarbeitete Fassung, die klarer macht, dass sich der zweite Satz allgemein auf nicht-standardisierte Funktionen bezieht:

```suggestion
    "de": "Diese Funktion ist nicht standardisiert. Wir raten davon ab, nicht-standardisierte Funktionen auf produktiven Webseiten zu verwenden, da sie nur von bestimmten Browsern unterstützt werden und sich in Zukunft ändern oder entfernt werden können. Unter Umständen kann sie jedoch eine geeignete Option sein, wenn es keine standardisierte Alternative gibt.",
```

---

Ok with me, an improvement on the original. It is a fine line between a smooth reading experience and full-on "Beamtendeutsch" 😀